scripts/serveBuild.js

```diff
@@ -10,11 +10,33 @@ const port = process.env.PORT || 3000;
 
 // Use the same proxy as webpack-dev-server during development (npm start)
 setupProxy(app);
+
 app.use(compression());
+const cfg = `
+  default-src 'self';
```

Comment:
Kun je deze waardes uit de NginX `.conf` file halen?

Reply:
dit is het geval!

Reply:
ik bedoel `const cfg = file.open('file.conf').read().extractTheRightPart();`